.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Have you seen this? https://github.com/pypi/warehouse/blob/main/.github/ISSUE_TEMPLATE/feature-request.md?plain=1
It's a bit shorter